ajax/libs/convnetjs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "convnetjs",
+  "filename": "convnet-min.js",
+  "version": "0.3.0",
+  "description": "Deep Learning in Javascript. Train Convolutional Neural Networks (or ordinary ones) in your browser.",
+  "keywords": [
+    "machine",
+    "learning",
+    "AI",
+    "convnet"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/karpathy/convnetjs.git"
+  },
+  "license": "MIT",
+  "homepage": "https://cs.stanford.edu/people/karpathy/convnetjs/",
+  "author": "Andrej Karpathy <andrej.karpathy@gmail.com>",
+  "npmName": "convnetjs",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
```

Comment:
I think we only need `convnet.js` and `convnet-min.js` because we can see https://github.com/karpathy/convnetjs#getting-started and http://cs.stanford.edu/people/karpathy/convnetjs/started.html to find out when the author talked about the library, he only mentioned the above two files.